js/app.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+
+function getVader() {
+  document.getElementById('person4Name').innerHTML = JSON.parse(this.response).name;
+}
+
+function getTatooine() {
+  document.getElementById('person4HomeWorld').innerHTML = JSON.parse(this.response).name
+}
+let vader = new XMLHttpRequest();
+vader.addEventListener("load", getVader);
+vader.open("GET", "https://swapi.co/api/people/4/");
+vader.send();
+
+let tatooine = new XMLHttpRequest();
+tatooine.addEventListener("load", getTatooine)
+tatooine.open("GET", "https://swapi.co/api/planets/1/");
+tatooine.send();
+
+function getSolo() {
+  document.getElementById('person14Name').innerHTML = JSON.parse(this.response).name;
+}
+
+function getHuman() {
+  document.getElementById('person14Species').innerHTML = JSON.parse(this.response).name;
+}
+let solo = new XMLHttpRequest();
+solo.addEventListener('load', getSolo);
+solo.open("GET", "https://swapi.co/api/people/14/");
+solo.send();
+
+let human = new XMLHttpRequest();
```

Comment:
Similar to the above note, you shouldn't hardcode the species but get it from the response from the previous call to get Han Solo.